packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -11,8 +11,10 @@ import {
 } from '@stencil/core'
 
 /**
- * @part wrapper - the modal wrapper overlay
- *
+ * @part wrapper - the modal wrapper overlay ! DEPRECATED IN FAVOR OF CONTAINER !
+ * @part container - the modal container
```

Comment:
I think all of these will be needed since they all reference something that is in the shadow dom, and this version of modal doesn't take any slots.